hack/render-candidate-catalog.sh

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env bash
+
+# render-candidate-catalog.sh creates a file-based OLM catalog containing the following:
+#
+# 1. The candidate shipwright-operator image (from a local/CI build)
+# 2. The existing Tekton and Shipwright operators that are released on OperatorHub.io
+#
+# The core of the catalog are JSON manifests contained in `test/catalog`.
+# This script copies those manifests to _output/catalog, then does a bit of opm and sed
+# manipulation to create catalog entries for the candidate operator deployment.
+
+set -e
+set -u
+set -o pipefail
+
+YQ_BIN=${YQ_BIN:-${PWD}/bin/yq}
+OPM_BIN=${OPM_BIN:-${PWD}/bin/opm}
+SED_BIN=${SED_BIN:-sed}
+
+channelSpec="_output/catalog/candidate/shipwright-operator-channel-candidate.json"
+bundleSpec="_output/catalog/candidate/shipwright-operator-bundle.json"
+
+# Initialize the new catalog
+rm -f "_output/catalog.Dockerfile"
+rm -rf "_output/catalog"
+cp -r test/catalog _output/
+
+# Replace the placeholder bundle name for the "candidate" OLM subscription channel
+${SED_BIN} -i -E 's|"name": "shipwright-operator-latest"|"name": "shipwright-operator.v'"${VERSION}"'"|g' "${channelSpec}"
```

Comment:
I think you refer here to `CSV_VERSION` as specified in Makefile:290.

```suggestion
${SED_BIN} -i -E 's|"name": "shipwright-operator-latest"|"name": "shipwright-operator.v'"${CSV_VERSION}"'"|g' "${channelSpec}"
```